library-core/src/main/java/com/mikepenz/fastadapter/adapters/ItemAdapter.java

```diff
@@ -466,7 +466,7 @@ public void remapMappedTypes() {
      * ItemFilter which extends the Filter api provided by Android
      * This calls automatically all required methods, just overwrite the filterItems method
      */
-    public class ItemFilter extends Filter {
+    private class ItemFilter extends Filter {
```

Comment:
this one should also be public (as you are allowed to provide your own `ItemFilter`